lib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ if (SWIFT_SWIFT_PARSER)
   list(TRANSFORM SWIFT_SYNTAX_MODULES PREPEND "SwiftSyntax::"
        OUTPUT_VARIABLE SWIFT_SYNTAX_TARGETS)
 
-  set(SWIFT_SYNTAX_LIBRARIES_SOURCE_DIR
+  set(SWIFT_SYNTAX_LIBRARIES_BUILD_DIR
       "${SWIFT_PATH_TO_EARLYSWIFTSYNTAX_BUILD_DIR}/lib/swift/host")
```

Comment:
This is really meant to be a "build" directory where earlyswiftsyntax's `.so` files exist.